punish/punish.py

```diff
@@ -608,6 +664,10 @@ def check(m):
         else:
             role = discord.utils.get(server.roles, name=default_name)
 
+        remove_roles = self.json.get(server.id, {}).get("REMOVE_ROLE_LIST")
```

Comment:
This doesn't need to be in setup if you use `get("REMOVE_ROLE_LIST", [])` in the other places.